Engine/Module/ModuleEditor.cpp

```diff
@@ -26,57 +36,323 @@ bool ModuleEditor::CleanUp()
 	return true;
 }
 
-void ModuleEditor::OpenScene() const
+void ModuleEditor::OpenScene(const std::string &path) const
 {
-	const char* path = "./Assets/Scenes/scene.scene";
-	std::ifstream ifs(path);
-	std::string serialized_scene_string(
-		(std::istreambuf_iterator<char>(ifs)),
-		(std::istreambuf_iterator<char>())
-	);
+	size_t readed_bytes;
+	std::string serialized_scene_string = App->filesystem->Load(path.c_str(), readed_bytes);
```

Comment:
I need to change this, but for now you will need to do a free of the data returned by this method